src/main/java/ca/corefacility/bioinformatics/irida/ria/web/users/UsersController.java

```diff
@@ -91,7 +91,7 @@ public UsersController(UserService userService, ProjectService projectService,
 	 * @return The name of the page.
 	 */
 	@RequestMapping
-	@PreAuthorize("hasRole('ROLE_ADMIN')")
+	@PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_MANAGER')")
```

Comment:
Another change to add in this file please.  The original issue was that it was throwing a `500` when access was denied.  Can you remove the `handleAccessDenied` method at the bottom of this file?  That's what was causing the 500 error as the template it's trying to load doesn't exist.  If a user tries to access this page it'll give the access denied page as it should.

Reply:
Removed in 1845c17

Reply:
Thanks.  This looks good.  I'll merge and push the release.